camel-k-loader-groovy/src/main/groovy/org/apache/camel/k/loader/groovy/dsl/ComponentConfiguration.groovy

```diff
@@ -49,13 +52,15 @@ class ComponentConfiguration {
         }
 
         if (!PropertyBindingSupport.build().withCamelContext(component.camelContext).withTarget(component).withProperty(name, value).bind()) {
+            LOG.error("Cannot set the component {} property {}", component.class.getName(), name)
```

Comment:
think we do not need to log things if we thrown an exception

Reply:
I will remove the log things in my next commit.

Reply:
Just removed the log.